pkgs/tools/audio/m4b-tool/default.nix

```
@@ -0,0 +1,47 @@
+{ fetchurl,
+  makeWrapper,
+  mp4v2,
+  php,
+  stdenv,
+
+  # By default m4b-tool will try to use the lbfdk_aac encoder (which cannot
+  # be built by default with ffmpeg due to licensing) but will fall back to
+  # ffmpeg's own aac encoder and issue a warning about the resulting audio
+  # quality. To silence this warning override ffmpeg with fdk-enabled ffmpeg
+  # e.g. ffmpeg-full.override { fdkaacExtlib = true; nonfreeLicensing = true; }
+  ffmpeg,
+
+  fdkAacSupport ? true, fdk-aac-encoder,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "m4b-tool";
+  version = "0.4.2";
+
+  src = fetchurl {
+    url = "https://github.com/sandreas/m4b-tool/releases/download/v.0.4.2/m4b-tool.phar";
```

```suggestion
    url = "https://github.com/sandreas/m4b-tool/releases/download/v.{version}/m4b-tool.phar";
```

Otherwise auto updates are broken.